src/Button/Button.Styled.js

```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components';
+
+
+export const StyledButton = styled.button`
+    color: #F6F9FC;
+    font:14px 'Arial';
+    border: none;
+    background-color:#FF4E54;
```

Comment:
@KBhautik6 read color from theme

Reply:
Ok, I will make new theme file for this and import on button.style.js and use that ::root file at here and from next time i will take of it.